lib/features/public_asset/domain/model/public_assets_in_identity_arguments.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:equatable/equatable.dart';
+import 'package:tmail_ui_user/features/public_asset/presentation/public_asset_controller.dart';
+
+class PublicAssetsInIdentityArguments with EquatableMixin {
+  final String htmlSignature;
+  final List<PublicAssetId> oldPublicAssetIds;
```

Comment:
can you more details on the name of `old` and `new`?

Reply:
names updated